app/Services/GameService.php

```diff
@@ -8,7 +8,7 @@ class GameService {
 
     function getGamesById (int $id)
     {
-        $games = $this->getGames();
+        $games = Game::find($id);
```

Comment:
this is all you should need, you can remove the foreach loop logic below. In order to make sure a NotFoundHttpException is thrown, i would suggest using the `findOrFail()` method instead.

Reply:
Resolved!